CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required (VERSION 3.17)
+cmake_policy (SET CMP0053 NEW)
+cmake_policy (SET CMP0054 NEW)
+
+project (
+        GOCART
+        VERSION 10.12.5
```

Comment:
Start with 1.0.0.   The version from the GCM GridComp will rapidly diverge.   No really good option, but this is the convention we have used with other repos that split off.

And to be fair, you should probably call it 0.9.0 for now while we are getting things in place.

Reply:
...